src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public static void main(String[] args) {
```

Comment:
Consider avoiding methods longer than 30 lines to improve readability. [See here.](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)